src/output-virtual.c

```diff
@@ -93,3 +99,32 @@ output_virtual_remove(struct server *server, const char *output_name)
 		}
 	}
 }
+
+void
+output_virtual_update_fallback(struct server *server)
+{
+	struct wl_list *layout_outputs = &server->output_layout->outputs;
+	const char *fallback_output_name = getenv("LABWC_FALLBACK_OUTPUT");
+
+	if (wl_list_empty(layout_outputs) && !string_null_or_empty(fallback_output_name)) {
+		wlr_log(WLR_DEBUG, "adding fallback output %s", fallback_output_name);
+		assert(!fallback_output);
```

Comment:
I am actually not sure about this assert.. It might be possible for a user to trigger this by disabling the fallback output via wlr-randr or similar. The wlroots layout might have a handler for that that removes it from the layout. Need to verify.

Reply:
Replaced the assert by adding it to the condition instead, this should make the logic more robust.